doc/xcodebuild.txt

```diff
@@ -1,31 +1,139 @@
 
 ==============================================================================
-CONTENTS                               *xcodebuild-contents*
+CONTENTS                                                         *XCB-Contents*
 
+      1. About.......................... |XCB-About|
+      2. Usage ......................... |XCB-Usage|
+        2.1  ............................. |XBuild|
+        2.2  ............................. |XTest|
+        2.3  ............................. |xcpretty|
+      3. Configuration ................. |XCB-Configuration|
+        3.1 .............................. |xcodebuild_run_command|
+        3.2 .............................. |xcodebuild_xcpretty_flags|
+        3.3 .............................. |xcodebuild_xcpretty_testing_flags|
 
 ==============================================================================
-INTRODUCTION                           *xcodebuild*
+ABOUT (1)                                                           *XCB-About*
 
-Xcodebuild.vim is a thin wrapper around Apple's xcodebuild command line
-application that lets you build and test your Xcode projects from within vim.
+`xcodebuild.vim` is a thin wrapper around Apple's `xcodebuild` command line
+application that lets you build and test your Xcode projects from within Vim.
 It dynamically figures out your project information and passes the proper
-flags to xcodebuild.
+flags to `xcodebuild`.
+
+If you have `xcpretty`[1] installed, `xcodebuild.vim` will use it to reformat
+the output from `xcodebuild`.
+
+This plugin was written by Gordon Fontenot[2]. Bugs and feature requests are
+welcomed, and can be posted on the GitHub repo[3].
 
-If you have xcpretty installed, it will use that to reformat the output from
-xcodebuild. By default, xcodebuild.vim passes the `--color` flag to xcpretty.
+[1]: https://github.com/supermarin/xcpretty
+[2]: http://gordonfontenot.com
+[3]: https://github.com/gfontenot/vim-xcodebuild
 
 ==============================================================================
-CONFIGURATION                             *xcodebuild-configuration*
+USAGE (2)                                                           *XCB-Usage*
+
+`xcodebuild.vim` provides two commands, one for building the project, and one
+for testing the project. It dynamically looks at your project configuration
+and determines which flags to pass to `xcodebuild`.
+
+Currently, `xcodebuild.vim` only looks at the first scheme that it finds in
+the `xcodeproj` file in the working directory. It uses the build settings in
+that scheme to determine which sdk to test against. If you have an
+`xcworkspace` file that contains your project (as is common with projects
+using CocoaPods[4]), it will build the scheme through the workspace.
+
+[4]: https://cocoapods.org/
+
+------------------------------------------------------------------------------
+                                                                       *XBuild*
+2.1 XBuild~
```

Comment:
It...highlights it different? I'm sure there's a semantic reason for this. Again, stole this from the legendary @christoomey, so I'm not sure.